pages/common/gh-issue.md

```diff
@@ -5,28 +5,28 @@
 
 - Display a specific issue:
 
-`gh issue view {{issue_number}}`
+`gh issue view {{1..n}}`
```

Comment:
This was fine as it was. `issue_number` is more descriptive of what is actually required than formal bounds.

Reply:
It can't be a mandatory change, it's incorrect suggestion to revert range placeholders back. I've just enhanced the description, now it is: `- Display an issue with a specific number:`.